Directory.Build.props

```diff
@@ -154,6 +154,7 @@
   <Import Project="eng\Dependencies.props" />
   <Import Project="eng\PatchConfig.props" />
   <Import Project="eng\ProjectReferences.props" />
+  <Import Project="src\Framework\Microsoft.AspNetCore.App.props" />
```

Comment:
Not the greatest location for common properties. Why not keep it in eng?

Reply:
Wasn't really thinking about that - i just left the file where it was. I'll move to eng/...that's a better place for it.